std/range/package.d

```diff
@@ -8665,10 +8665,11 @@ struct NullSink
     void put(E)(E){}
 }
 
+///
 @safe unittest
 {
     import std.algorithm : map, copy;
-    [4, 5, 6].map!(x => x * 2).copy(NullSink());
+    [4, 5, 6].map!(x => x * 2).copy(NullSink()); // data is discarded
```

Comment:
Actually I made git pull --rebase before further modifications and I did not touch this file. Its even same as previous version in my fork - anyone knows how to fix this is PR?